client/my-sites/plans/plan-overview/plan-feature/index.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import CompactCard from 'components/card/compact';
+
+const PlanFeature = ( { button, description, heading } ) => {
+	return (
+		<CompactCard className="plan-features__feature">
```

Comment:
The CSS here also needs to be updated. I think we can omit the second instance of `feature` in all of these cases now.